docs/tutorial/queries.rst

```diff
@@ -4,77 +4,82 @@
 =========
 
 After the database and schema are set up, we can add some actual data.
-For example, let's add "Blade Runner 2049" to the database. It's possible to
-add movie, director and actor data all at once:
+For this tutorial we will use the command-line REPL tool to do that,
+so let's start it up:
+
+.. code-block:: bash
+
+    $ edgedb
```

Comment:
This should be `edgedb -I myapp` (or whatever instance name we choose for the tutorial).